### PR TITLE
feat: Enhance documentation

### PR DESCRIPTION
**Pull request by GuiLeme**
### What does this PR do? 

It just enhance the documentation.

### CODE REVIEW

**Review by Arthurk12 — APPROVED** [review]
Well done!

**Comment by PadaliyaSavan88** [comment]
Must awaited, waiting for pull request to get merged.

File: README.md [hunk 1]
```diff
@@ -34,18 +77,36 @@ The `fetch-dependencies` script is responsible for cloning the `bigbluebutton/h5
...
 npm run build-bundle
 ```
 
-The above command will generate the `dist` folder, containing the bundled JavaScript file named `H5pPlugin.js`. This file can be hosted on any HTTPS server.
+The above command will generate the `dist` folder containing the bundled JavaScript file named `H5pPlugin.js`. This file can be hosted on any HTTPS server.
+
+Preferebly, that bundled js file is placed into the Bigbluebutton's assets portion of the server, so for that, do the following: 
+
+```bash
+mkir -p /var/www/bigbluebutton-default/assets/plugins/plugin-h5p/
```
Comment (PadaliyaSavan88) [inline README.md]: Change:  mkir to mkdir

Comment (TiagoJacobs) [inline README.md]:
```suggestion
mkdir -p /var/www/bigbluebutton-default/assets/plugins/plugin-h5p/
```

**Pull request merged**